### DISCUSSION TITLE
Beginner Contributors

### DISCUSSION THREAD

**Discussion by Monukumar193** [github/india#837] [GitHub Constellation 🇮🇳 2022 Q&A]
Hello folks, I am Monu Kumar BE Computer Science Student. I am freshmen here i want to be start contribute in open source. Please provide me the path. I am really very excited to learn this and contribute open source all over the and want to connect the amazing community. Love the GitHub

**Comment by kranurag7** [comment]
@Monukumar193 You can get started here. https://github.com/explore 
Try looking for projects that interests you, and then look for good first issues in the project. You can also contribute to the open source project that you use on a daily basis if the project is open source. Also try joining communities and the members there will guide you in getting started and getting familiar with the project.

**Reply by Monukumar193** [reply]
Thanks for initiate me🙏🙏

**Comment by dhlodestar** [comment]
👋 @Monukumar193.. that's a great question! To contribute to Open Source, first pick your areas of interest and discover projects on [GitHub.com](http://github.com/) that match these. You can then start exploring open issues in these projects where you can add value - either by fixing a bug, by enhancing the project, or keeping documentation up-to-date etc. After this, fork the project->make your changes->open a PR to merge your changes back.
For more info, check these out:
[Finding ways to contribute to GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
[The Open Source guide for contributions](https://opensource.guide/how-to-contribute/)
[How to contribute to an Open Source Project](https://dev.to/codesphere/how-to-start-contributing-to-open-source-projects-on-github-534n)

**Reply by Monukumar193** [reply]
Thank you so much sir😊

**Comment by richakumar** [comment]
https://github.com/open-source is also a good place to start

**Reply by Monukumar193** [reply]
Thanks😊

**Comment by FRTMentor** [comment]
Please register in the FRT - https://futurereadytalent.in/
Complete you learning and get hands on internship experience.
Get evaluated by IT experts and get certified.

**Comment by RijoKsd** [comment]
Hi,

I'm Rijo Sebastian, a Computer Science graduate with a specialization in MERN stack development from Kerala, India. I'm passionate about open-source projects and eager to collaborate with like-minded individuals. 
Contact me on LinkedIn https://www.linkedin.com/in/rijoksd 
Check out my portfolio https://rijoksd.netlify.app.

**Comment by Raffi-stang-seher** [comment]
I’m learning Arduino and also practicing my English. I’m looking for simple game project ideas. Maybe we can brainstorm together?
